doc/users/next_whats_new/set_colors_ticks_and_ticklabels_rcparams.rst

```diff
@@ -0,0 +1,7 @@
+The color of ticks and tick labels can be set independently from rcParams
+-------------------------------------------------------------------------
+
+The color of ticks and tick labels can now be set independently from the
+matplotlib rc file or the rcParams using the xtick or ytick.color and
+.labelcolor parameters.
```

Comment:
We have custom extension so 
```rst
:rc:`xtick.labelcolor`
```
will render nicely in the output, could you use that here?

Can this text include an example of how to use this?  I'm not sure our average user will understand how to use this new feature from this text.

Reply:
Of course! I have updated the text. Let me know if it is better.